src/Scene/NodeProcess.js

```diff
@@ -276,47 +268,37 @@ function updateNodeElevation(quadtree, node, elevationLayers) {
     // Here we only make sure that the tile with level == levelElevation == 11 has its elevation texture.
     // Also see TileMesh.setTextureElevation
     var tileNotDownscaled = (node.level === node.levelElevation) ?
-        node :
+        null :
         node.getParentLevel(node.levelElevation);
```

Comment:
`node.getParentLevel(node.levelElevation)` it is not necessarily the ancestor not down scaled,
it may be less than `node.levelElevation`. If there's big zoom (level 2 to 20), it's not necessary to load all `levelElevation`.